### PR TITLE
Feature/fix text effect in tweet template

### PR DESCRIPTION
**Pull request by mahaelbasuony**
Screenshot 2021-09-08 at 21 41 05

### CODE REVIEW

**Review by medhatdawoud — CHANGES_REQUESTED** [review]
Screenshot 2021-09-08 at 10 46 29 PM

this is not matching the design for the small numbers, it should work in both sizes for all numbers including single or double numbers and `+99` as well

**Comment by medhatdawoud** [comment]
> Screenshot 2021-09-08 at 10 46 29 PM
> 
> this is not matching the design for the small numbers, it should work in both sizes for all numbers including single or double numbers and `+99` as well

ignore this , it is for another PR

File: style.css [hunk 1]
```diff
@@ -470,40 +499,51 @@ ul {
  font-size: 14px;
 }
 
-.tweet-template__icon{
+.tweet-template__icon::before{
```
Comment (medhatdawoud) [inline style.css]: why did you add `::before` here?

doesn't make sense, if should be there only with the `content: ' '` attribute as well to attach something to the element

**Comment by mahaelbasuony** [comment]
> > Screenshot 2021-09-08 at 10 46 29 PM
> > this is not matching the design for the small numbers, it should work in both sizes for all numbers including single or double numbers and `+99` as well
> 
> ignore this , it is for another PR

okay ... can you check this PR for merge it 👍

**Review by medhatdawoud — APPROVED** [review]

**Pull request merged**